### DISCUSSION TITLE
No max height of images makes the experience not so pleasant

### DISCUSSION THREAD

**Discussion by kristoffermagma** [community/community#61223] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

Videos have a max height, but images don't. So when we upload tall images (which is often the case for screenshots from the phone) it becomes quite hard to overview. 
https://github.com/community/community/assets/92582030/e7af2a42-a1ab-4d80-a1ca-de52b4aeff3e

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by DevBrent** [comment]
This is frankly unacceptable that Microsoft allows this issue to persist. It's literally one line of CSS that will fix this issue across the board. All developers know how to right click "View Image" when they want more. I definitely don't need a 2688 pixel tall image every time my iPhone users file a bug report. GitHub Issues is terrible for images.
